ui/src/layout/Findings/Malware/MalwareTable.js

```diff
@@ -17,7 +17,14 @@ const MalwareTable = () => {
             id: "filePath",
             sortIds: ["findingInfo.path"],
             accessor: "findingInfo.path",
-            width: 300
+            width: 200
+        },
+        {
+            Header: "Rule name",
+            id: "ruleName",
+            sortIds: ["findingInfo.ruleName"],
+            accessor: "findingInfo.ruleName",
+            width: 200
         },
```

Comment:
I think it would probably be better to have this after Malware Name before Path, the rule is serving a similar purpose to Rule Name at the moment. (This will probably change if we redo the model anyway though so we can leave it if we like)